src/cli.rs

```diff
@@ -832,8 +832,9 @@ fn init_commands<'a, C: CLI<'a>>() -> Vec<Command<'a, C>> {
                 let records = keystore
                     .records()
                     .await
-                    .filter(|rec| rec.ro.asset_def.code == asset.item && match &account {
-                        Some(address) => rec.ro.pub_key.address() == address.0,
+                    .into_iter()
+                    .filter(|rec| rec.record_opening().asset_def.code == asset.item && match &account {
```

Comment:
`rec.asset_definition().code` (or we could even add an `asset_code()` function to `Record`)

Reply:
Added asset_code() to `Record`